jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +132,213 @@ async def coroutine():
+            return
+
+        if not ssa:
+            self.app.show_message(_(common_strings.error), _("No SSA was entered"), tobefocused=dialog.buttons[0])
+            return
+
+        dialog.close()
+        self.set_center_frame()
+
+        client_creation_data = self._build_registration_payload(parsed_iss, ssa)
+
+        asyncio.ensure_future(self._create_client_coroutine(client_creation_data, parsed_iss, ssa))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)